scripts/render.js

```diff
@@ -236,6 +236,7 @@ async function compile(page) {
 
     const tasks = languages.map(async lang => {
         const affiliate_language_code = common.getAffiliateSignupLanguage(lang);
+        const deriv_language_code = lang === 'EN' ? '' : `${lang.toLowerCase().replace(/_/g, '-')}/`;
```

Comment:
do we still need this or redirect from /en to / is done in deriv.com?

Reply:
deriv.com now handles that, but it overwrites the whole like which overwrites the affiliate details provided with the url